ios-base/Classes/APIManager.h

```diff
@@ -1,17 +1,28 @@
 //
 //  APIManager.h
-//  ios-base
-//
-//  Created by Danis Ziganshin on 14.02.14.
-//  Copyright (c) 2014 FlatStack. All rights reserved.
 //
 
 #import <Foundation/Foundation.h>
+#import <AFNetworking/AFNetworking.h>
+#import <AFNetworking/AFNetworkActivityIndicatorManager.h>
+
+@class AFHTTPRequestOperationManager;
 
+#pragma mark -
 @interface APIManager : NSObject
 
 + (APIManager*)sharedManager;
 
-- (void)getCurrentDateWithCompleteBlock:(ObjectCallback)block;
+@property (nonatomic, strong, readonly) AFHTTPRequestOperationManager *operationManager;
 
 @end
+
+
+#pragma mark -
+@interface APIManager (Date)
```

Comment:
I do not like this way of separating functions to different classes. @NikolaevSergey what do you think?